README.md

```diff
@@ -1 +1,56 @@
-# Test-Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# M
+
+> One paragraph statement about the project.
+
+![screenshot](screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://busiwa24.github.io/Test-Portfolio/))
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+-You only need a web browser on your local machine
+
+
+
```

Comment:
- [ ] These texts are coming from the template. You should update these with proper.